docs/_includes/forms/inc-input-large.html

```diff
@@ -1,4 +1,4 @@
 
 <div class="ds44-form__container">
-    <label for="name2" class="ds44-formLabel"><span class="ds44-labelTypePlaceholderLarge ds44-moveLabel">{{ include.label }}{{ include.requiredStar }}</span> <input type="text" id="name2" class="ds44-inpLarge" {{ include.required }} /></label>
+    <label for="largeInp{{ include.number }}" class="ds44-formLabel"><span class="ds44-labelTypePlaceholderLarge ds44-moveLabel">{{ include.label }}{{ include.requiredStar }}</span> <input type="text" id="largeInp{{ include.number }}" class="ds44-inpLarge" {{ include.required }} /><button class="ds44-reset"><i class="icon icon-cross icon--large" aria-hidden="true"></i><span class="visually-hidden">effacer le contenu saisi</span></button></label>
```

Comment:
Est ce que tu peux utiliser "id" au lieu de "number" stp ? Ça a plus de sens car c'est bel et bien un id puisqu'on le met dans l'attribut "id".
Et piur être cohérent avec l'autocomplete etc, est ce que tu pourrais modifier `largeInp{{ include.number }}` en `largeInp-{{ include.id }}` stp ?

Reply:
Si tu veux, mais ce "number" me sert à construire d'autres choses que des IDs, c'est pourquoi j'ai choisi un terme générique (je peux donc l'associer à un ID et à un FOR qui lui est relié).